src/Pipedrive.net/Converters/CustomFieldConverter.cs

```diff
@@ -193,95 +193,98 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
-                            writer.WriteValue(dr.EndDate);
-                            break;
-                        case AddressCustomField a:
-                            writer.WritePropertyName(field.Key);
-                            writer.WriteValue(a.Value);
-                            writer.WritePropertyName($"{field.Key}_subpremise");
-                            writer.WriteValue(a.Subpremise);
-                            writer.WritePropertyName($"{field.Key}_street_number");
-                            writer.WriteValue(a.StreetNumber);
-                            writer.WritePropertyName($"{field.Key}_route");
-                            writer.WriteValue(a.Route);
-                            writer.WritePropertyName($"{field.Key}_sublocality");
-                            writer.WriteValue(a.Sublocality);
-                            writer.WritePropertyName($"{field.Key}_locality");
-                            writer.WriteValue(a.Locality);
-                            writer.WritePropertyName($"{field.Key}_admin_area_level_1");
-                            writer.WriteValue(a.AdminAreaLevel1);
-                            writer.WritePropertyName($"{field.Key}_admin_area_level_2");
-                            writer.WriteValue(a.AdminAreaLevel2);
-                            writer.WritePropertyName($"{field.Key}_country");
-                            writer.WriteValue(a.Country);
-                            writer.WritePropertyName($"{field.Key}_postal_code");
-                            writer.WriteValue(a.PostalCode);
-                            writer.WritePropertyName($"{field.Key}_formatted_address");
-                            writer.WriteValue(a.FormattedAddress);
-                            break;
-                        case OrganizationCustomField org:
-                            writer.WritePropertyName(field.Key);
-                            writer.WriteValue(org.Value);
-                            break;
-                        case PersonCustomField p:
-                            writer.WritePropertyName(field.Key);
-                            writer.WriteValue(p.Value);
-                            break;
-                        case UserCustomField u:
-                            writer.WritePropertyName(field.Key);
-                            writer.WriteValue(u.Value);
-                            break;
+                        writer.WritePropertyName(field.Key);
+                        writer.WriteNull();
+                    }
+                    else
+                    {
+                        switch (field.Value)
+                        {
+                            case StringCustomField s:
+                                writer.WritePropertyName(field.Key);
+                                writer.WriteValue(s.Value);
+                                break;
+                            case IntCustomField i:
```

Comment:
Don't override my work. I've replaced `IntCustomField` with `LongCustomField` in 2c50bfd9ac39f9721664415cfa8eb27c81c575ca

Reply:
My apologies, as I forgot to include that update in this particular file.